queries/liquidsoap/folds.scm

```diff
@@ -7,8 +7,9 @@
  (if)
  (if_condition)
  (if_then)
+ (elsif_condition)
+ (elseif_then)
```

Comment:
I don't see why there's a for `if_condition` and `elsif_condition` to be this specific? Isn't it still a normal expression?. Also typo probably :D

Reply:
Typo fixed, thanks.

The condition and body of each `then` clause can be full expressions with bindings, same expressions as the body of a function.
```liquidsoap
if
  x = f()
  x or g()
then
...
end

Reply:
Well that's a surprising thing to learn then.